simulation/behavior_tree_plugin/src/pedestrian/pedestrian_action_node.cpp

```diff
@@ -73,14 +73,17 @@ auto PedestrianActionNode::estimateLaneletPose(const geometry_msgs::msg::Pose &
      * @note Hard coded parameter. 1.0 is a matching threshold for lanelet.
```

Comment:
Invalid comment

Reply:
Thanks, I fixed.

Reply:
I confirmed in https://github.com/tier4/scenario_simulator_v2/pull/1194/commits/d0bfdf62a14f6e2da2a78259ccffbd42c840d375